ci/docker/Dockerfile.build.ubuntu_gpu_cu100

```diff
@@ -67,10 +67,6 @@ RUN /work/ubuntu_docs.sh
 COPY install/ubuntu_tutorials.sh /work/
 RUN /work/ubuntu_tutorials.sh
 
-ENV CUDNN_VERSION=7.3.1.20
-COPY install/ubuntu_cudnn.sh /work/
-RUN /work/ubuntu_cudnn.sh
```

Comment:
?

Reply:
On my previously PR, I resolved the commit by keeping two cudnn installation accidentally... let seperate this part and create another one to fix it

Reply:
I created another PR #14996 to clean up the duplicate cudnn installation